src/api/app/views/webui/configuration/index.html.haml

```diff
@@ -38,6 +41,80 @@
         = f.label(:default_tracker) do
           Default Tracker
         ~ f.collection_select :default_tracker, IssueTracker.all, :name, :description, { include_blank: true }, class: 'form-select'
+      .mb-3
+        - ymp_url = CONFIG['ymp_url']
+        = f.label(:ymp_url) do
+          YMP URL
+          %small.form-text URL prefix for one-click installation files
+          - unless ymp_url.nil?
+            %small.form-text.text-danger= inherit % ['ymp_url']
+        = f.text_field :ymp_url, value: !ymp_url.nil? ? ymp_url : @configuration.ymp_url, disabled: !ymp_url.nil?, class: 'form-control'
+      .mb-3
+        - bugzilla_host = CONFIG['bugzilla_host']
+        = f.label(:bugzilla_url) do
+          Bugzilla URL
+          %small.form-text Default bugzilla instance for reporting to bugowners
+          - unless bugzilla_host.nil?
+            %small.form-text.text-danger= inherit % ['bugzilla_host']
+        = f.text_field :bugzilla_url, value: !bugzilla_host.nil? ? bugzilla_host : @configuration.bugzilla_url,
+                                      disabled: !bugzilla_host.nil?, class: 'form-control'
+      .mb-3.form-check
+        - allow_anonymous = CONFIG['allow_anonymous']
+        = f.check_box :anonymous, checked: !allow_anonymous.nil? ? allow_anonymous : @configuration.anonymous,
+                                  disabled: !allow_anonymous.nil?, class: 'form-check-input'
+        = f.label(:anonymous, class: 'form-check-label') do
+          Anonymous
+          %small.form-text Allow anonymous browsing of this OBS instance.
+          - unless allow_anonymous.nil?
+            %small.form-text.text-danger= inherit % ['allow_anonymous']
+      .mb-3.form-check
+        - new_user_registration = CONFIG['new_user_registration']
+        = f.check_box :registration, checked: !new_user_registration.nil? ? new_user_registration : @configuration.registration,
```

Comment:
I'm not sure what values this could take... but it doesn't seem a boolean: https://github.com/openSUSE/open-build-service/blob/827272c9222aaf702704518b02b714d497e1d177/src/api/app/models/configuration.rb#L135

From the review app, I cannot mark this field as enabled.

Reply:
Fixed